docs/index.Rmd

```diff
@@ -567,8 +583,17 @@ Grouping can be done by:
 
 * single column name
 * seq of column names
-* map of keys (group names) and row indexes
 * value returned by function taking row as map (limited to `:select-keys`)
+* map of keys (arbitrary group names) to sequences of row indexes
+
+In the case of the first three of these methods, each sub-dataset contains all and only rows from the original data set that share the same grouping value:
+
+* the value of the row in a specified single column
+* the sequence of resulting values for a sequence of column names
+* the value returned by the function taking row as map
+
+In the case of the map from group names to sequences of indexes, each sub-dataset will contain all and only rows with the indexes in one value sequence of indexes.
```

Comment:
This sounds incomprehensible a little bit to me:

_[...] with the indexes in one value sequence of indexes_

Maybe something like that?

_[...] with the indexes listed in the sequence for a given group name (a key)._

Reply:
I see your point.  :-)  I like your revision and will make the change.